### DISCUSSION TITLE
How do I do a PR if I want to add something in branch 4.1?

### DISCUSSION THREAD

**Discussion by astridx** [joomla/joomla-cms#32973] [Q&A]
Example: I have implemented something in branch 4.0 because it builds on things that are not yet integrated in 4.1. If I set the PR against branch 4.1, I cannot see what I have changed because many things are missing and therfore are many differnces.
https://github.com/joomla/joomla-cms/pull/32969 and https://github.com/joomla/joomla-cms/pull/32968

**Comment by richard67** [comment]
One way could be to make the PR against 4.0 and later when the 4.1-dev branch has been updated to 4.0-dev so the changes mentioned above went up to 4.1-dev, what should happen one day, rebase the PR to 4.1-dev.

Another way could be to contact @bembelimen and check with him what's the best way.
